app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id'
+  has_many :likes
+  has_many :comments
```

Comment:
- [ ] To correctly match the associations between models  according to the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png), I think it would be better if you also add the `foreign_key` on both `:likes` and `:comments` table. i.e.
```ruby
has_many :posts, foreign_key: 'author_id'
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'
```

Reply:
@tresorsawasawa There seems to be a discrepancy in the diagrams. I don't think this change is necessary
- Diagram from the project - [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png)
- Diagram you attached in your review - [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png)

Reply:
Hi @newhoteng 

I really apologizing to you. After the call we did, I noticed that the link of the[ project's requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project_v1_1.md) given to me it seems to have been a discrepancy. 
You are true accoreding to your project's requirements.